lib/index.js

```diff
@@ -0,0 +1,126 @@
+const fs = require('fs');
+const {
+  fetchLabels,
+  createLabels,
+  deleteLabels,
+  updateLabels,
+} = require('./api');
+
+async function labelControl() {
+  const labelData = await fetchLabels();
+  const formattedPublishedLabels = await labelFormatter(labelData);
+  const data = fs.readFileSync('./label-config.json', { encoding: 'utf8' });
+
+  const localConfig = JSON.parse(data.toString());
+  const illegalLabels = checkForSameLabelName(
+    localConfig,
+    formattedPublishedLabels
+  );
+  if (illegalLabels) {
+    throw new Error(
+      'The config has labels with the same name. The Label names have to be unique.'
+    );
+  }
+  const newLabels = checkForNewOrOldLabels(
+    localConfig,
+    formattedPublishedLabels
+  );
+  const labelsToDelete = checkForNewOrOldLabels(
```

Comment:
Hmm I'm a bit confused about using the same function for getting the labels to be removed.

Reply:
Yes, maybe the naming is a bit confusing but at the end the logic is the same for both cases...we have to compare the ids to see if there are new or old ids left. We just have to change the output make it work for both cases 
🤔